openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -198,7 +197,7 @@ def create_account_with_params(request, params):
     # Announce registration
     REGISTER_USER.send(sender=None, user=user, registration=registration)
 
-    create_comments_service_user(user)
+    # create_comments_service_user(user)
```

Comment:
lets add a comment here as to why we have commented this code.

Reply:
Done